### PR TITLE
Feature/add prefix tables function

### PR DESCRIPTION
**Pull request by MikeVanMourik**
# Describe your changes

it was requested we get a function for retrieving the prefixes used in wiser tables
in the asana ticket for the wts cleanup

## Type of change

Please check only ONE option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How was this tested?

ran this locally

# Checklist before requesting a review
- [x] I have reviewed and tested my changes
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I selected `develop` as the base branch and not `main`, or the pull request is a hotfix that needs to be done directly on `main`
- [x] I double checked all my changes and they contain no temporary test code, no code that is commented out and no changes that are not part of this branch
- [x] I added new unit tests for my changes if applicable

# Related pull requests

# Link to Asana ticket
https://app.asana.com/0/1204371236183532/1206141964915217/f

### CODE REVIEW

File: GeeksCoreLibrary/Core/Services/WiserItemsService.cs [hunk 1]
```diff
@@ -3761,6 +3761,30 @@ public async Task<List<WiserItemFileModel>> GetItemFilesAsync(IWiserItemsService
 
             return result;
         }
+        
+        /// <inheritdoc />
+        public async Task<List<string>>GetDedicatedTablePrefixesAsync()
```
Comment (GilianJuice) [inline GeeksCoreLibrary/Core/Services/WiserItemsService.cs:3766]: PLease add a space before the function name.

**Review by GilianJuice — CHANGES_REQUESTED** [review]
This seems to give compilation errors, please fix them:
image

File: GeeksCoreLibrary/Core/Services/WiserItemsService.cs (hunk 1 above)
Comment (MikeVanMourik) [reply GeeksCoreLibrary/Core/Services/WiserItemsService.cs]: done

**Comment by MikeVanMourik** [comment]
> This seems to give compilation errors, please fix them: image

not sure why those changes were not complete, i think i might have reverted something before i comitted, resolved now

**Pull request merged**